install-dev/data/xml/tab.xml

```diff
@@ -5,450 +5,697 @@
                     </tab>
                 <tab id="Module_Catalog" id_parent="Modules" active="1" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminParentModulesCatalog</class_name>
+                    <wording>Module Catalog</wording>
+                    <wording_domain>Admin.Navigation.Menu</wording_domain>
                 </tab>
                     <tab id="Module_Page_Catalog" id_parent="Module_Catalog" active="1" enabled="1" hide_host_mode="0" icon="">
                         <class_name>AdminModulesCatalog</class_name>
+                        <wording>Module Catalog</wording>
+                        <wording_domain>Admin.Navigation.Menu</wording_domain>
                     </tab>
                     <tab id="Marketing_tools" id_parent="Module_Catalog" active="1" enabled="1" hide_host_mode="0" icon="">
                         <class_name>AdminAddonsCatalog</class_name>
+                        <wording>Module Selections</wording>
+                        <wording_domain>Admin.Navigation.Menu</wording_domain>
                     </tab>
                 <tab id="Module_Page_old" id_parent="Modules" active="0" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminModules</class_name>
+                    <wording/>
+                    <wording_domain/>
                 </tab>
 
             <!--LOOK & FEEL-->
             <tab id="Look_feel" id_parent="IMPROVE" active="1" enabled="1" hide_host_mode="0" icon="desktop_mac">
                 <class_name>AdminParentThemes</class_name>
+                <wording>Design</wording>
+                <wording_domain>Admin.Navigation.Menu</wording_domain>
             </tab>
 
                 <tab id="Themes" id_parent="Look_feel" active="1" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminThemes</class_name>
+                    <wording>Theme &amp; Logo</wording>
+                    <wording_domain>Admin.Navigation.Menu</wording_domain>
                 </tab>
                 <tab id="Themes_Catalog" id_parent="Look_feel" active="1" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminThemesCatalog</class_name>
+                    <wording>Theme Catalog</wording>
+                    <wording_domain>Admin.Navigation.Menu</wording_domain>
                 </tab>
                 <tab id="Mail_Theme_Parent" id_parent="Look_feel" active="1" enabled="1" hide_host_mode="0" icon="">
-                  <class_name>AdminParentMailTheme</class_name>
+                    <class_name>AdminParentMailTheme</class_name>
+                    <wording>Email Theme</wording>
+                    <wording_domain>Admin.Navigation.Menu</wording_domain>
                 </tab>
                     <tab id="Mail_Theme" id_parent="Mail_Theme_Parent" active="1" enabled="1" hide_host_mode="0" icon="">
-                      <class_name>AdminMailTheme</class_name>
```

Comment:
Indentation in the whole file is totally broken :sweat_smile:

Reply:
Yes because the structure makes no sense. This indentation is meant to reflect the menu structure which doesn't follow the xml structure (or is it the other way around? 🤪) 

Anyway I spent some hours trying to fix it and I found myself falling through the "refactor ALL the things!" rabbit hole, so I backtracked and decided to focus on the original intent.